client/src/components/Buttons.js

```diff
@@ -36,3 +36,30 @@ export const StartDashboardButton = withStyles({
     marginTop: '30px'
   },
 })(Button);
+
+
+export const InterviewDetailButtonHighlighted = withStyles({
```

Comment:
Is there anyway we can re-use buttons and just have different labels on them? The reason for cohesive styling is so we can re-use as much code as possible - i.e. one button with it's own state that chooses color based on whether it's toggled on or off